.taskcluster.yml

```diff
@@ -32,34 +32,16 @@ tasks:
       then: ${event.pull_request.body}
 
     taskboot_image: "mozilla/taskboot:0.4.2"
-  in:
-    $if: 'tasks_for == "github-push" || (tasks_for == "github-pull-request" && event["action"] in ["opened", "reopened", "synchronize"])'
-    then:
-      - taskId: { $eval: as_slugid("lint_task") }
-        created: { $fromNow: "" }
-        deadline: { $fromNow: "1 hour" }
-        provisionerId: proj-bugbug
-        workerType: batch
-        payload:
-          maxRunTime: 3600
-          image: python:3.10
-          command:
-            - "/bin/bash"
-            - "-lcx"
-            - "git clone --quiet ${repository} &&
-              cd bugbug &&
-              git -c advice.detachedHead=false checkout ${head_rev} &&
-              pip install --disable-pip-version-check --no-cache-dir --progress-bar off -r test-requirements.txt &&
-              pre-commit run -a --show-diff-on-failure"
-        metadata:
-          name: bugbug lint
-          description: bugbug lint
-          owner: mcastelluccio@mozilla.com
-          source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
-      - $if: 'tasks_for == "github-push"'
-        then:
-          taskId: { $eval: as_slugid("version_check_task") }
+  in:
+    $let:
+      default_task_definition:
```

Comment:
this is the only change. 
Additionally I think its possible to create a base task_definition with all items (not just owner and source). But I think that might be hard to debug incase something goes wrong ?